docs/pvacseq/getting_started.rst

```diff
@@ -20,3 +20,63 @@ The example data output can be reproduced by running the following command:
    -e 8,9,10
 
 A detailed description of all command options can be found on the :ref:`Usage <pvacseq_run>` page.
+
+.. _pvacseq_docker:
+
+Running pVACseq using Docker
+____________________________
+
+A pVACtools Docker image is available on DockerHub using the
+``griffitlab/pvactools`` tag. After `installing Docker <https://docs.docker.com/install/>`_
```

Comment:
Would it be useful to link this to <https://hub.docker.com/r/griffithlab/pvactools>?